subsys/bluetooth/lib/encrypted_ad_data/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2023 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_ENCRYPTED_AD_DATA
```

Comment:
It would probably make sense to make this follow the API naming as well, i.e. `config BT_EAD`.

Reply:
but then it's harder to search for using the online kconfig browser or `menuconfig`

Reply:
Is it? Doesn't the search there work on the Kconfig symbol description and not the symbol name?

Reply:
My bad it does. Searching for "Bluetooth" does give the `BT` symbol that has the description set to "Bluetooth".
I agree then.